recipes/mne-python_1.3/construct.yaml

```diff
@@ -74,6 +74,8 @@ specs:
   - mne-ari =0.1.2
   - mne-kit-gui =1.0.1
   - mne-icalabel =0.4  # [not win]
+  - mnelab =0.8.5
+  - pyside6 =6.4.1
```

Comment:
We may be able to remove the version pinning (or that entire dependency altogether, as it should get installed implicitly when requesting mnelab)